src/loc.ml

```diff
@@ -85,6 +101,14 @@ let print ppf loc =
           Format.fprintf pp "%s\n%*s\n" line
             stop_c
             (String.make len '^')
+        else
```

Comment:
`dune` files are usually pretty small, but I believe that we want to avoid printing very large chunks. What do you think about adding a check so that if the error spans more than a given number of line, we don't print anything? I'd say 10 lines is a good number to start from.

Reply:
10 seems like a reasonable number to start with.